python-client/giskard/slicing/slice.py

```diff
@@ -69,13 +160,73 @@ def __str__(self) -> str:
         operator = "<=" if self.equal else "<"
         return f"`{self.column}` {operator} {_pretty_str(self.value)}"
 
+    def init_code(self):
+        return f"{self.__class__.__module__}.{self.__class__.__name__}({repr(self.column)}, {repr(self.value)}, equal={repr(self.equal)})"
+
+    def to_clause(self):
+        return {
+            'columnName': self.column,
+            'comparisonType': 'LOWER_THAN_EQUALS' if self.equal else "LOWER_THAN",
+            'value': self.value
+        }
+
 
 class EqualTo(ComparisonClause):
     _operator = operator.eq
 
     def __str__(self) -> str:
         return f"`{self.column}` == {_pretty_str(self.value)}"
 
+    def to_clause(self):
+        return {
+            'columnName': self.column,
+            'comparisonType': 'IS',
+            'value': self.value
+        }
+
+
+class NotEqualTo(ComparisonClause):
+    _operator = operator.ne
+
+    def __str__(self) -> str:
+        return f"`{self.column}` != {_pretty_str(self.value)}"
+
+    def to_clause(self):
+        return {
+            'columnName': self.column,
+            'comparisonType': 'IS_NOT',
+            'value': self.value
+        }
+
+
+def generate_clause(clause: Dict[str, str]) -> Clause:
+    if clause['comparisonType'] == 'IS':
+        return EqualTo(clause['columnName'], clause['value'])
+    elif clause['comparisonType'] == 'IS_NOT':
+        return NotEqualTo(clause['columnName'], clause['value'])
+    elif clause['comparisonType'] == 'CONTAINS':
+        return ContainsWord(clause['columnName'], clause['value'], is_not=False)
+    elif clause['comparisonType'] == 'DOES_NOT_CONTAINS':
+        return ContainsWord(clause['columnName'], clause['value'], is_not=True)
+    elif clause['comparisonType'] == 'STARTS_WITH':
+        return StartsWith(clause['columnName'], clause['value'])
+    elif clause['comparisonType'] == 'ENDS_WITH':
+        return EndsWith(clause['columnName'], clause['value'])
+    elif clause['comparisonType'] == 'GREATER_THAN_EQUALS':
+        return GreaterThan(clause['columnName'], clause['value'], equal=True)
+    elif clause['comparisonType'] == 'GREATER_THAN':
+        return GreaterThan(clause['columnName'], clause['value'])
+    elif clause['comparisonType'] == 'LOWER_THAN_EQUALS':
+        return LowerThan(clause['columnName'], clause['value'], equal=True)
```

Comment:
Same here, why add a custom parameter and not just return LowerThan OR EqualTo ? Would make functions do only what their name says